### PR TITLE
162633502 Fix image uploads

### PR DESCRIPTION
**Pull request by mubstimor**
### What does this PR do?
Changes image upload process for articles.

#### Tasks to be completed.
* Remove Pillow as a dependency
* Store path to full image on article

### CODE REVIEW

**Comment by araaliFarooq** [comment]
LGTM

**Comment by mycok** [comment]
LGTM

**Comment by NabunyaLilian** [comment]
LGTM

File: authors/apps/articles/admin.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
 from django.contrib import admin
 
-from .models import Article, ArticleImage, Tag
+from .models import Article, Tag
 
-class ArticleImageInline(admin.TabularInline):
-    model = ArticleImage
-    extra = 3
+# class ArticleImageInline(admin.TabularInline):
```
Comment (lym) [inline authors/apps/articles/admin.py]: Why not just delete the class?

Comment (mubstimor) [reply authors/apps/articles/admin.py]: resolved

**Pull request merged**